smoke-test/tests/domains/domains_test.py

```diff
@@ -21,6 +33,39 @@ def test_healthchecks(wait_for_healthchecks):
 @pytest.mark.dependency(depends=["test_healthchecks"])
 def test_create_list_get_domain(frontend_session):
 
+    # Construct upstream tables.
+    upstream_tables: List[UpstreamClass] = []
+    upstream_table_1 = UpstreamClass(
+        dataset=builder.make_dataset_urn("bigquery", "upstream_table_1", "PROD"),
+        type=DatasetLineageTypeClass.TRANSFORMED,
+    )
+    upstream_tables.append(upstream_table_1)
+    upstream_table_2 = UpstreamClass(
+        dataset=builder.make_dataset_urn("bigquery", "upstream_table_2", "PROD"),
+        type=DatasetLineageTypeClass.TRANSFORMED,
+    )
+    upstream_tables.append(upstream_table_2)
+
+    # Construct a lineage object.
+    upstream_lineage = UpstreamLineage(upstreams=upstream_tables)
+
+    # Construct a MetadataChangeProposalWrapper object.
+    lineage_mcp = MetadataChangeProposalWrapper(
+        entityType="dataset",
+        changeType=ChangeTypeClass.UPSERT,
+        entityUrn=builder.make_dataset_urn("bigquery", "downstream"),
+        aspectName="upstreamLineage",
+        aspect=upstream_lineage,
+        systemMetadata=
```

Comment:
@jjoyce0510 looks like there's some code missing here? Seems to be causing test failures

Reply:
Thanks! Addressing